tests/behaviour/concept/thing/BUILD

```diff
@@ -4,6 +4,7 @@
 
 load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
 load("@rules_rust//rust:defs.bzl", "rust_test")
+# TODO: Add test_ prefix to target names
```

Comment:
Haven't done not to have conflicts with the guys working on things, might do now